services/state/schedule/schedule.proto

```diff
@@ -33,116 +39,167 @@ option java_multiple_files = true;
 
 /**
  * Representation of a Hedera Schedule entry in the network Merkle tree.
- *
- * As with all network entities, a schedule has a unique entity number, which is usually given along
- * with the network's shard and realm in the form of a shard.realm.number id.
  */
```

Comment:
This doesn't seem to conform to the Style Guidelines - "Include a line break (<br/>) after the first line, then a brief description, if appropriate. Not every field needs additional description, but every message does."

Reply:
Updated.